### PR TITLE
Added tests for TableName

### PR DESCRIPTION
**Pull request by EinKrebs**

### CODE REVIEW

File: storage/clickhousespanstore/tablename_test.go [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTableName_AddDbName(t *testing.T) {
+	assert.Equal(t, TableName("database_name.table_name_local"), TableName("table_name_local").AddDbName("database_name"))
+}
+
+func TestTableName_ToGlobal(t *testing.T) {
```
Comment (pavolloffay) [inline storage/clickhousespanstore/tablename_test.go:13]: `ToGlobal` has maybe too ambiguous definition. Maybe it's better to define `ToLocal` and always add `_local` suffix.

Comment (EinKrebs) [reply storage/clickhousespanstore/tablename_test.go:13]: Done in #67

**Comment by EinKrebs** [comment]
Gonna rewrite tests after merging #69

File: storage/clickhousespanstore/tablename_test.go [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestTableName_AddDbName(t *testing.T) {
+	assert.Equal(t, TableName("database_name.table_name_local"), TableName("table_name_local").AddDbName("database_name"))
+}
+
+func TestTableName_ToLocal(t *testing.T) {
+	var tableName TableName = "some_table"
```
Comment (pavolloffay) [inline storage/clickhousespanstore/tablename_test.go:14]: assign the value directly

```
tableName :=
```

**Review by pavolloffay — APPROVED** [review]

**Pull request merged**